resources/public/api/conf/1.0/application.raml

```diff
@@ -1,6 +1,6 @@
 #%RAML 1.0
 ---
-title: MTD Template
+title: Submit an End of Period Statement
```

Comment:
needs to be title of microservice e.g. Individual Losses (MTD) or Individuals Charges (MTD)

Reply:
Updated